src/functionalTest/java/uk/gov/hmcts/divorce/caseworker/CaseworkerAddBailiffReturnFT.java

```diff
@@ -0,0 +1,43 @@
+//package uk.gov.hmcts.divorce.caseworker;
```

Comment:
Why is the entire class commented out?

Reply:
As above, we've still a ton of clean-up to complete. Need to get Master branch into usable if messy state, currently Master is broken.